webapp/src/main/java/com/box/l10n/mojito/service/security/user/UserService.java

```diff
@@ -206,7 +262,7 @@ public User updatePassword(String currentPassword, String newPassword) {
    * @return The newly created user
    */
   public User createUserWithRole(String username, String password, Role role) {
-    return createUserWithRole(username, password, role, null, null, null, false);
+    return createUserWithRole(username, password, role, null, null, null, null, true, false);
```

Comment:
If the user role is a basic user I think we should set the canTranslateAllLocales boolean to false, so something like

`createUserWithRole(username, password, role, null, null, null, null, role != ROLE_USER ? true : false , false);`